dashboard/test/lib/expired_deleted_account_pii_scrubber_test.rb

```diff
@@ -0,0 +1,90 @@
+require 'test_helper'
+
+class ExpiredDeletedAccountPiiScrubberTest < ActiveSupport::TestCase
+  include Minitest::RSpecMocks
+
+  let(:described_class) {ExpiredDeletedAccountPiiScrubber}
+  let(:described_instance) {described_class.new(options)}
+  let(:options) {{}}
+  let(:user) {create(:user, :deleted)}
+  let(:older_than_ttl_date) {(User::SOFT_DELETED_USER_TTL + 1.day).ago}
+  let(:newer_than_ttl_date) {(User::SOFT_DELETED_USER_TTL - 1.day).ago}
+
+  describe '#scrub_pii_from_expired_deleted_accounts!' do
+    subject(:scrub_pii) {described_instance.scrub_pii_from_expired_deleted_accounts!}
+
+    before do
+      user.update(deleted_at: older_than_ttl_date)
+      Cdo::Metrics.stubs(:push)
+      ChatClient.stubs(:message)
+      Honeybadger.stubs(:notify)
+    end
+
+    it 'should run the PII scrub service on expired deleted accounts' do
```

Comment:
Please avoid using "should" when describing test cases: https://www.betterspecs.org/#should
```suggestion
    it 'runs PII scrub service on expired deleted accounts' do
```